### ISSUE TITLE
Poor readability of editor warnings

### ISSUE THREAD

**Issue by MaddyGuthridge** [DanielMSchmidt/zed-synthwave#9]
This is very hard to read:

<img width="716" height="216" alt="Image" src="https://github.com/user-attachments/assets/041b0e40-63c8-436a-945c-3fccfd171d40" />

Generally, the standard minimum contrast ratio for text is 4.5:1. This text has a ratio of 1.26:1, which is not readable for most users.
